styles/variables-carbon.less

```diff
@@ -58,6 +58,7 @@
 @agate-popup-max-width: 100%;
 @agate-popup-border-width: 1px 0;
 @agate-popup-border-radius: 0;
+@agate-popup-fullscreen-border-radius: @agate-popup-border-radius;
```

Comment:
I'm not sure this is needed. @stanca-pop-lgp could you please let me know about this?